src/test/java/org/aerogear/digger/client/services/JobServiceTest.java

```diff
@@ -86,18 +87,29 @@ public void shouldCreateJobWithCredentials() throws Exception {
     @Test
     public void shouldUpdateJobWithCredentials() throws Exception {
         UsernamePasswordCredential repoCredential = new UsernamePasswordCredential();
+        BuildDiscarder buildDiscarder = new BuildDiscarder();
         repoCredential.setId("testCredentialId");
         repoCredential.setUsername("test");
         repoCredential.setPassword("test");
-        jobService.update(server, "name", "repo", "branch", repoCredential, null);
+        jobService.update(server, "name", "repo", "branch", buildDiscarder, repoCredential, null);
         verify(server, times(1)).deleteCredential(repoCredential.getId(), false);
         verify(server, times(1)).createCredential(repoCredential, false);
         verify(server, times(1)).updateJob(anyString(), anyString());
     }
 
+    @Test
     public void shouldDeleteJob() throws Exception {
         jobService.delete(server, "testJob", null);
         verify(server, times(1)).deleteCredential(anyString(), anyBoolean());
         verify(server, times(1)).deleteJob(anyString());
     }
+
+    @Test
+    public void shouldSetValidBuildDiscardValue() {
+        BuildDiscarder buildDiscarder = new BuildDiscarder();
+        buildDiscarder.setStoreArtifactsDays(-4);
+        buildDiscarder.setStoreArtifactsTotal((5));
+        assert(buildDiscarder.getStoreArtifactsDays() == -1);
+        assert(buildDiscarder.getStoreArtifactsTotal() == 5);
```

Comment:
instead of builtin assert (which is used for something else originally), please use AssertJ's `assertThat()` function. Example: https://github.com/aerogear/digger-java/blob/16d13e60b83331fb131be45a93f17850ce778480/src/test/java/org/aerogear/digger/client/services/BuildServiceTest.java#L119-L119